Objective-C/CBLReplicatorConfiguration.m

```diff
@@ -132,13 +136,24 @@ - (void) setChannels: (NSArray<NSString *> *)channels {
 }
 
 
+#if TARGET_OS_IPHONE
+- (void) setAllowReplicatingInBackground: (BOOL)allowReplicatingInBackground {
+    [self checkReadonly];
+    
+    if (_allowReplicatingInBackground != allowReplicatingInBackground) {
```

Comment:
The `if` doesn't accomplish anything; just set the variable. Actually, the same is true of all the other setter methods.

Reply:
I will remove all that.